### PR TITLE
Gradio for Tabular Data Science Workflows Guide

### PR DESCRIPTION
**Pull request by merveenoyan**
# Description

This is the PR for Gradio for data science workflows guide. It includes three examples:
- Building a tabular interface for tabular ML model from scratch
- Building a data dashboard
- Using skops integration

The below checklist doesn't really apply (but let me know if there's anything I need to pay attention to)

cc: @freddyaboulton @adrinjalali
# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: guides/5)other_tutorials/using_gradio_for_tabular_workflows.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+## Using Gradio for Tabular Data Science Workflows
+
+Related spaces: https://huggingface.co/spaces/scikit-learn/gradio-skops-integration, https://huggingface.co/spaces/scikit-learn/tabular-playground, https://huggingface.co/spaces/merve/gradio-analysis-dashboard
+
+
+## Introduction
+
+Tabular data science is the most widely used domain of machine learning, with problems ranging from customer segmentation to churn prediction. Communicating your work to stakeholders of your project or clients in various stages of tabular data science workflow is very important and can sometimes be cumbersome, preventing data scientists from focusing on what matters, data analysis and model building. Data scientists can end up spending hours building a dashboard that takes in dataframe and returning plots, or returning a prediction or plot of clusters in a dataset. In this guide, we'll go through how to use `gradio` to improve your data science workflows. We will also talk about how to use `gradio` and `skops` to build interfaces with only one line of code!
```
Comment (adrinjalali) [inline guides/5)other_tutorials/using_gradio_for_tabular_workflows.md:8]: make `skops` a hyperlink?

**Review by adrinjalali — APPROVED** [review]
LGTM

File: guides/5)other_tutorials/using_gradio_for_tabular_workflows.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+## Using Gradio for Tabular Data Science Workflows
+
+Related spaces: https://huggingface.co/spaces/scikit-learn/gradio-skops-integration, https://huggingface.co/spaces/scikit-learn/tabular-playground, https://huggingface.co/spaces/merve/gradio-analysis-dashboard
+
+
+## Introduction
+
+Tabular data science is the most widely used domain of machine learning, with problems ranging from customer segmentation to churn prediction. Communicating your work to stakeholders of your project or clients in various stages of tabular data science workflow is very important and can sometimes be cumbersome, preventing data scientists from focusing on what matters, data analysis and model building. Data scientists can end up spending hours building a dashboard that takes in dataframe and returning plots, or returning a prediction or plot of clusters in a dataset. In this guide, we'll go through how to use `gradio` to improve your data science workflows. We will also talk about how to use `gradio` and [skops](https://skops.readthedocs.io/en/stable/) to build interfaces with only one line of code!
```
Comment (dawoodkhan82) [inline guides/5)other_tutorials/using_gradio_for_tabular_workflows.md:8]: Nit (run-on sentence): Communicating your work to stakeholders of your project or clients in various stages of tabular data science workflow is very important and can sometimes be cumbersome, preventing data scientists from focusing on what matters, data analysis and model building.

Maybe this reads better:
Throughout various stages of the tabular data science workflow, communicating your work to stakeholders or clients can be cumbersome; which prevents data scientists from focusing on what matters, such as data analysis and model building.

**Comment by dawoodkhan82** [comment]
Left two quick comments, otherwise looks great! @merveenoyan

File: guides/5)other_tutorials/using_gradio_for_tabular_workflows.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+outputs = [gr.Dataframe(row_count = (2, "dynamic"), col_count=(1, "fixed"), label="Predictions", headers=["Failures"])]
+
+model = joblib.load("model.pkl")
+
+def infer(input_dataframe):
+  return pd.DataFrame(model.predict(input_dataframe))
+  
+
+gr.Interface(fn = infer, inputs = inputs, outputs = outputs, examples = [[df.head(2)]]).launch()
```
Comment (freddyaboulton) [inline guides/5)other_tutorials/using_gradio_for_tabular_workflows.md:33]: The variable `df` is not defined in this code snippet which may confuse readers. Can you add the code that defines it here?

**Review by freddyaboulton — APPROVED** [review]
Awesome guide! Thanks @merveenoyan

**Review by dawoodkhan82 — APPROVED** [review]

**Comment by merveenoyan** [comment]
Can you merge this if you approve so I can announce this :')

**Pull request merged**